staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_test.go

```diff
@@ -407,5 +408,5 @@ func testSetup(t *testing.T, opts ...setupOption) (context.Context, *Cacher, tea
 		t.Fatalf("Failed to inject list errors: %v", err)
 	}
 
-	return ctx, cacher, terminate
+	return ctx, cacher, server, terminate
```

Comment:
Instead of returning it everywhere, let's just do:
```
func testSetupWithEtcdServer(...) {
  // what we have now
}

func testSetup(...) {
  ctx, cacher, _ terminate := testSetupWithEtcdServer()
  return ctx, cacher, terminate
}
```

Reply:
Done.